todoman/cli.py

```diff
@@ -224,49 +247,48 @@ def edit(ctx, id, todo_properties, interactive):
             setattr(todo, key, value)
 
     if interactive or (not changes and interactive is None):
-        ui = TodoEditor(todo, ctx.obj['db'].lists(), ctx.obj['formatter'])
+        ui = TodoEditor(todo, ctx.db.lists(), ctx.ui_formatter)
```

Comment:
You can use `todo edit` *partially* interactively as well, e.g. use `--due` to override that field, but also use `-i` to launch the editor afterwards (with the new values)